codalab/lib/spec_util.py

```diff
@@ -106,3 +107,10 @@ def is_dashboard(name):
 
 def is_public_home(name):
     return name == 'home'
+
+
+def is_academic_email(email):
```

Comment:
This doesn't really belong in `spec_util`, which is the utility for specifications...you can just keep it in bundle_model for now.

Also, the logic we want is anything that ends with .edu or .edu.*, not just .edu.cn.

Reply:
Thanks Percy! I'm really confused where to put this `is_academic_email ` function, and I will leave it in `bundle_model` for now.